rules/S6640/csharp/metadata.json

```
@@ -0,0 +1,17 @@
+{
+  "title": "Allowing unsafe code is security-sensitive",
```

To me "usafe code" is a bit too generic. What about "memory unsafe"?

```suggestion
  "title": "Allowing memory unsafe code is security-sensitive",
```